apps/server/src/apps/server.app.ts

```diff
@@ -29,6 +31,28 @@ import {
 	createAndStartPrometheusMetricsAppIfEnabled,
 } from './helpers/prometheus-metrics';
 
+const createAndStartAdminApiServer = async (logger: Logger) => {
+	const nestAdminServerExpress = express();
```

Comment:
Starting a 3. express app on a single pod sounds strange. We speak about it, but i prefare that you import server modules and start it on seperate deployment.

Reply:
We can do this in a separate deployment, it will require an additional work in the deployment area (additional secrets to be set up etc.) and also it will mean that two separate deployments will connect to the same database (which I believe is not recommended to have a proper logic separation between the deployments), but we can do that of course. Please maybe just confirm this decision with @Metauriel as it will require additional workload and we have to make sure it's final and conscious.